src/styles/app/45-local-connections.css

```
@@ -74,99 +74,42 @@
-	--local-connections-hover-halo: color-mix(
-		in srgb,
-		var(--interactive-accent) 28%,
-		transparent
-	);
-	--local-connections-hover-halo-soft: color-mix(
-		in srgb,
-		var(--interactive-accent) 12%,
-		transparent
-	);
-
 	width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)